ow_plexil/src/plans/TestPanTilt.plp

```diff
@@ -0,0 +1,62 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Simple plan to test Pan and Tilt functions
+
+#include "plan-interface.h"
+
+TestPanTilt: Sequence
```

Comment:
Best to omit 'Sequence' since it's the default, and an alias for CheckedSequence.  Specifying the latter or UncheckedSequence is fine if you want to be explicit about this.  I'm making a request to removed 'Sequence' from the language because it can be misleading.